pkg/network/msg.go

```diff
@@ -1,6 +1,21 @@
 package network
 
-import "github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+)
+
+// MessageSerializer represents a interface for KNoT messages
+type MessageSerializer interface {
+	Serialize() ([]byte, error)
+}
+
+// Message represents a KNoT message
+type Message struct {
```

Comment:
should we export it?
```suggestion
type message struct {
```

Reply:
Done!